src/analysis/rMTA/MTA_MIQP.m

```diff
@@ -0,0 +1,141 @@
+function [v_res, solution] = MTA_MIQP (OptimizationModel, KOrxn, varargin)
+% Returns the OptimizationModel solution of a particular MTA problem and
+% an specific model
+%
+% USAGE:
+%
+%    [v_res, success, unsuccess] = MTA_MIQP (OptimizationModel, KOrxn, numWorkers, timeLimit, printLevel)
+%
+% INPUT:
+%    OptimizationModel:       Cplex Model struct
+%    KOrxn:            perturbation in the model (reactions)
+%    numWorkers:       number of threads used by Cplex.
+%    printLevel:       1 if the process is wanted to be shown on the
+%                      screen, 0 otherwise. Default: 1.
+%
+% OUTPUTS:
+%    Vout:             Solution flux of MIQP formulation for each case
+%    solution:         Cplex solution struct
+%
+% .. Authors:
+%       - Luis V. Valcarcel, 03/06/2015, University of Navarra, CIMA & TECNUN School of Engineering.
+% .. Revisions:
+%       - Luis V. Valcarcel, 26/10/2018, University of Navarra, CIMA & TECNUN School of Engineering.
+
+% Check the input information
+p = inputParser;
+% check requiered arguments
+addRequired(p, 'OptimizationModel');
+addRequired(p, 'KOrxn');
+% Check optional arguments
+addOptional(p, 'numWorkers', 0);
```

Comment:
Do you want them as optional arguments (requires the right positions) or as parameters?
Personally, I always prefer parameters, as they make later modifications of the header much easier without breaking backward compatability, and, especially with a larger number of arguments, they make (imo) reading the calls easier, as it is clearer, what each value means.